service-templates/docker-compose.yaml

```
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  
+  testrunner:
+    image: $DEV_IMAGE_TAG
+    build:
+      dockerfile: Dockerfile.dev
+      context: .
+      args:
+        OTP_VERSION: $OTP_VERSION
+        ## @SETUP-THRIFT Delete the following line if your service does not require thrift
+        THRIFT_VERSION: $THRIFT_VERSION
```

А нельзя ли такие вещи за логику в шаблонах спрятать, типа
```suggestion
        {{#needs_thrift}}
        THRIFT_VERSION: $THRIFT_VERSION
        {{/needs_thrift}}
```
?